qcodes/instrument_drivers/QDevil/QDevil_QDAC.py

```diff
@@ -938,35 +956,43 @@ def ramp_voltages(self, channellist, v_startlist, v_endlist, ramptime):
                             fast_vend=v_endlist, step_length=0.001,
                             slow_steps=1, fast_steps=steps)
 
-    def ramp_voltages_2d(self, slow_chans, slow_vstart, slow_vend,
-                         fast_chans, fast_vstart, fast_vend,
-                         step_length, slow_steps, fast_steps):
+    def ramp_voltages_2d(
+            self,
+            slow_chans: Sequence[int],
+            slow_vstart: Sequence[float],
+            slow_vend: Sequence[float],
```

Comment:
This was documented as List[int] but surely the voltages must be list of floats? This is also needed for functions calling this to check correctly

Reply:
Yes, the voltages are floats.